src/modules/peek/Peek.FilePreviewer/Previewers/IPreviewer.cs

```diff
@@ -12,6 +13,8 @@ public interface IPreviewer : INotifyPropertyChanged
     {
         bool IsPreviewLoaded { get; }
 
+        public static bool IsFileTypeSupported(string fileExt) => throw new NotImplementedException();
```

Comment:
I think this can be made abstract to not require an implementation.

Reply:
Hm, I had tried that but got this error: The interface 'IPreviewer' cannot be used as type argument. Static member 'IPreviewer.IsFileTypeSupported(string)' does not have a most specific implementation in the interface.

My C# is a bit rusty, but I think it might relate to this open issue: 
https://github.com/dotnet/csharplang/issues/4436